calmTheStorm/calmTheStorm/DisasterInfoData.swift

```diff
@@ -0,0 +1,72 @@
+//
+//  DisasterInfoData.swift
+//  calmTheStorm
+//
+//  Created by Student Account on 5/17/21.
+//
+
+import SwiftUI
+
+struct DisasterInfoItem: Identifiable {
+    var id = UUID()
+    var info: String
+}
+
+struct infoRow: View{
+    let whichInfo: DisasterInfoItem
+    
+    var body: some View{
+        HStack{
+            Text(" \(whichInfo.info)")
+                .font(.custom("Avenir", size: 18))
+        }
+    }
+}
+
```

Comment:
I think the data should be in a separate file.  I don't think its good practice to put the data in with the business logic.  Maybe create a separate file that stores the data arrays.

Reply:
For the next sprint, I will work on creating a separate page for the logic and clean things up on the code side of things.